File: JAVA/bubblesort_m1.java
import java.util.*;
class bubblesort_m1
{
  // method to print array
  public static void printarray(int arr[])
  {
  for(int i=0;i<arr.length;i++)
    {
      System.out.print(arr[i]+" ");
    }
    System.out.println();
  }
  // main method
  public static void main(String args[])
  {
    int a[]={8,4,2,9,1};
    int n=a.length;
    //dispaly unsorted array
    System.out.println("the unsorted array is:");
    //calling method
    printarray(a);
    //bubble sort
    for(int i=0;i<n-1;i++)
    {
      for(int j=0;j<n-1-i;j++)
      {
        if(a[j]>a[j+1])
        { 
           //swapping
         int temp=a[j];
         a[j]=a[j+1];
         a[j+1]=temp;
        }
        
      }
    }
    //display sorted array
    System.out.println("the sorted array is:");
    //calling method
    printarray(a);
    
  }
}

File: C /bubblesort-m1.c
#include <stdio.h>
void main()
{
    int a[5]={5,4,1,2,6};
    printf("the unsorted elements are:\n");
    for (int i = 0; i < 5; i++)
    {
        printf("%d ", a[i]);
    }
    printf("\nThe sorted elements are:\n");
    for (int i = 0; i < 5 - 1; i++)
    {
        for (int j = 0; j < 5 - 1 - i; j++)
        {
            if (a[j] > a[j + 1])
            {   
                int temp = a[j];
                a[j] = a[j + 1];
                a[j + 1] = temp;
            }
        }
    }
    for (int i = 0; i < 5; i++)
    {
        printf("%d ", a[i]);
    }
}